ajax/libs/videojs-playLists/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "videojs-playLists",
+  "filename": "videojs-playlists.min.js",
+  "version": "0.2.0",
+  "description": "Playlists done right for Videojs",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jgallen23/videojs-playLists"
+  },
+  "keywords": [
+    "videojs",
+    "tags",
```

Comment:
Not sure why we need this `tags` keyword?
Could you tell me?
Because I can't find the keyword from upstream.
Thank you.

Reply:
I apologize, I copy and pasted the package.json file from another one I worked on, and I must have forgotten that one.

Reply:
@sufuf3 I also was playing around with the desktop version of github and I used that to send my commit message.

Reply:
Thanks :)